api/urls.py

```diff
@@ -11,8 +20,10 @@
 
 # These come after /api/..
 urlpatterns = [
+    path("feeds/", feeds_pagination),
     path("feeds/<str:feed_type>/<str:attack_type>/<str:age>.<str:format_>", feeds),
     path("enrichment", enrichment_view),
+    path("generalhoneypot", general_honeypot_view),
```

Comment:
I suggest to rename this endpoint and split the two words "general" and "honeypot" with - or _